exercise/staff_views.py

```diff
@@ -202,6 +205,58 @@ def get_common_objects(self):
         )
 
 
+class ExamSessionEdit(UpdateView):
```

Comment:
Why is `ExamSessionEdit` in `exercise/staff_views.py` instead of exammode views?

Reply:
Move to exammode/views or exammode/staff_views

Reply:
This has not been done yet.

Reply:
This is now committed